pom.xml

```
@@ -11,6 +11,17 @@
         available with a single command.
     </description>
     <url>https://heroku.github.com/webapp-runner</url>
+    <licenses>
+        <license>
+            <name>BSD-3-Clause</name>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <name>Manuel Fuchs</name>
+            <organization>salesforce.com, inc.</organization>
```

If we wanted the organisation name to be consistent with:
https://github.com/salesforce/oss-template/blob/271c27e41c8dc9cccb66e1f6e39dedb83a091678/LICENSE.txt#L3

xref:
https://github.com/salesforce/oss-template/commit/e09907e0909f79c1ab32fa36bb7faa7f650bcd55

```suggestion
            <organization>salesforce.com, inc.</organization>
```

Also, this probably needs updating at some point:
https://github.com/heroku/webapp-runner/blob/01ae855c4f45f3aeb558c7aa693381ef1517f25a/LICENSE#L1

---

I made it consistent with the other repos we have. Literally copied from the JVM function runtime. :) That said, the capital S seems to be correct. I'll fix.

---

Yeah, I've been tip-toeing around the copyright notice since I don't know what to do. @jsimone transferred this repo to us, but I think he technically still has the copyright. Maybe @jsimone can chime in with his 5c?